app/test_main.py

```diff
@@ -1,3 +1,42 @@
+import pytest
+from typing import Any
 from app.main import get_human_age
```

Comment:
![image](https://user-images.githubusercontent.com/104218550/220937766-cc38639d-4250-482f-9968-be479db92b78.png)

Reply:
still not fixed